content/millennium/r4/financial/general/charge-item.md

```diff
@@ -41,11 +41,13 @@ The following fields are returned if valued:
 * [Custom Attribute]
 * [Description]
 * [Modifier]
+* [National Drug Product]
 * [Net Price]
 * [Offset By]
 * [Performing Location]
 * [Priority]
 * [Procedure]
+* [Quantity Conversion Factor]
```

Comment:
Should these extensions be listed on the structure def page? https://github.com/cerner/fhir.cerner.com/blob/charge-item-r4-ndc-qcf-extensions/content/millennium/r4/foundation/conformance/structure-definition.md

Reply:
Added - https://github.com/cerner/fhir.cerner.com/pull/683/commits/0f794e324f3a6936c02184bb826f31097bb87313